components/EventCard/EventCard.tsx

```diff
@@ -28,9 +28,9 @@ const EventCard: React.FC<EventCardProps> = ({
         <Text style={styles.title}>{title}</Text>
         <Text>{date}</Text>
         <View style={styles.locationContainer}>
-          <Text style={styles.location}>{location}</Text>
-          <Text>{latitude}</Text>
-          <Text>{longitude}</Text>
+          <Text style={styles.location}>{location +  "  "}</Text>
+          <Text>{Number(latitude).toFixed(2) + "  "}</Text>
+          <Text>{Number(longitude).toFixed(2)}</Text>
```

Comment:
Good for now but I think we will need to use at least 3 decimal points since 2 decimal points gives us an error radius of 2.78 km I think.